src/boot_O2/loadfragment2.c

```diff
@@ -1,120 +1,120 @@
 #include "global.h"
 #include "system_malloc.h"
 
-UNK_TYPE4 D_80096C30 = 2;
-
-#ifdef NON_MATCHING
-// This needs lots of work. Mostly regalloc and getting the address of D_80096C30 placed in s5 at the beginning of the
-// function
-void Load2_Relocate(u32 allocatedVRamAddr, OverlayRelocationSection* overlayInfo, u32 vRamStart) {
-    s32 sectionLocations[4];
-    u32* regReferences[32];
-    u32 regValues[32];
+s32 gLoad2LogSeverity = 2;
+
+void Load2_Relocate(void* allocatedVRamAddr, OverlayRelocationSection* ovl, u32 vRamStart) {
```

Comment:
I think it's worth pointing out the difference between the two functions/files, it's confusing enough having two slightly different versions of the same file, only one of which is used, I think?

Reply:
Yeah, only Load2 is actually used. To be honest I don't really understand the differences between the two files, other than that Load2 is used and Load just exists.

Reply:
So looking at it closer, I noticed that the `Load` functions (the unused one in `loadfragment.c`) don't have `vRamEnd` arguments. I think its purpose was to dynamically determine `vRamEnd` and why it calculates `ovl->bssSize + size` and reallocates if `size` is to small.